app/javascript/gamesFactory.js

```diff
@@ -431,7 +513,7 @@ angular.module('dappChess').factory('games', function (crypto, navigation, gameS
           console.log('Could not send state and move to blockchain', e);
         }
       }
-    }, 10000);
+    }, 3000);
```

Comment:
Can you move this to a constant? I'd rather change this for production use later.